scripts/install

```diff
@@ -69,7 +69,7 @@ ynh_exec_as_app mitractl create-account $admin $adminpass admin
 # SETUP & START SERVICE
 #=================================================
 
-ynh_config_add_systemd --template=mitra
+ynh_config_add_systemd --template=mitra.service
```

Comment:
`systemd.service` will be renamed `mitra.service` by default, so `--template=mitra.service` is not needed

Reply:
Yeah, now there's no more template specified https://github.com/YunoHost-Apps/mitra_ynh/blob/bdcc96bdef8f933cce0ad1508118067cdef09957/scripts/install#L78C1-L78C23